pixi.toml

```diff
@@ -0,0 +1,275 @@
+  { task = "test", environment = "test-all-deps-py313" },
+]
+
+[environments]
+# Testing
+test-default = { features = ["test"] }
+test-all-but-numba = { features = [
+  "py313",
+  "test",
+  "backends",
+  "performance",
+  "dask",
+  "plotting",
+  "extras",
+] }
+test-all-but-dask = { features = [
+  "py312",
+  "test",
+  "backends",
+  "performance",
+  "numba",
+  "plotting",
+  "extras",
+] }
+test-all-deps-py313 = { features = [
+  "py313",
+  "test",
+  "backends",
+  "performance",
+  "numba",
+  "dask",
+  "plotting",
+  "extras",
+] }
+test-all-deps-py311 = { features = [
+  "py311",
+  "test",
+  "backends",
+  "performance",
+  "numba",
+  "dask",
+  "plotting",
+  "extras",
+] }
+
+test-all-mypy-py311 = { features = [
+  "py311",
+  "test",
+  "backends",
+  "performance",
+  "numba",
+  "dask",
+  "plotting",
+  "extras",
+  "mypy",
+] }
+
+test-all-mypy-py313 = { features = [
+  "py313",
+  "test",
+  "backends",
+  "performance",
+  "numba",
+  "dask",
+  "plotting",
+  "extras",
+  "mypy",
+] }
+
+test-bare-minimum = { features = ["test", "minimal"] }
+test-bare-min-and-scipy = { features = [
+  "test",
+  "minimal",
+  "minimum-scipy",
+] }
+
+# Extra
+typing = { features = ["mypy"] }
+doc = { features = [
+  "doc",
+  "backends",
+  "test",
+  "performance",
+  "plotting",
+  "extras",
+], solve-group = "doc" }
+pre-commit = { features = ["pre-commit"], no-default-feature = true }
+
+[package]
+name = "xarray"
+version = "dynamic" # dynamic versioning needs better support in pixi https://github.com/prefix-dev/pixi/issues/2923#issuecomment-2598460666 . Putting `version = "dynamic"` here for now until pixi recommends something else.
+
+[package.build]
+backend = { name = "pixi-build-python", version = "==0.4.0" }
+
+[package.host-dependencies]
+setuptools = "*"
+setuptools_scm = "*"
+
+[package.run-dependencies]
+python = "*"
+numpy = "*"
+pandas = "*"
+
+packaging = "24.1.*" #? Can be removed?
+git = "*"            # needed for dynamic versioning
+
+[dependencies]
+xarray = { path = "." }
+
+[target.linux-64.dependencies]
+pydap-server = "*"
+
+[feature.minimal.dependencies]
+# minimal versions
+python = "3.11.*"
+numpy = "1.26.*"
+pandas = "2.2.*"
+
+[feature.minimum-scipy.dependencies]
+scipy = "1.13.*"
+
+[feature.py311.dependencies]
+python = "3.11.*"
+
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.py313.dependencies]
+python = "3.13.*"
+
+[feature.backends.dependencies]
+# files
+h5netcdf = "*"
+h5py = "*"
+hdf5 = "*"
+netcdf4 = "*"
+zarr = "*"
+rasterio = "*"
+
+# opendap
+pydap = "*"
+lxml = "*"  # Optional dep of pydap
+
+# s3
+boto3 = "*"
+fsspec = "*"
+aiobotocore = "*"
+
+[feature.numba.dependencies]
+numba = "*"
+numbagg = "*"
+
+[feature.dask.dependencies]
+dask = "*"
+distributed = "*"
+
+[feature.performance.dependencies]
+flox = "*"
+bottleneck = "*"
+numexpr = "*"
+pyarrow = "*"
+opt_einsum = "*"
+
+[feature.performance.pypi-dependencies]
+jax = "*"
```

Comment:
`jax` isn't under `accel`. Can we put it under `extras` for now?

Also see https://github.com/pydata/xarray/issues/9826 for possible followup work.